docs/labs.md

```diff
@@ -138,3 +138,7 @@ This setting is (currently) _sticky_ to a user's session: it only takes effect w
 ## New room header & details (`feature_new_room_decoration_ui`) [In Development]
 
 Refactors visually the room header and room sidebar
+
+## Notifications panels (`feature_notifications_panel`)
+
+A list of all events notifying the current user. Only works with unencrypted content.
```

Comment:
> Only works with unencrypted content.

This isn't true, it depends on a user's push rules. If a user marked their `.m.room.encrypted*` push rules with highlight=true then it'd include those too.

Reply:
But in reality, does that ever happen? I'd be more inclined to add smething a bit simpler

Reply:
Its also not right because we don't only show `notifications` - we show only highlighting notifications. You will have more notifications than you will find in this panel even for unencrypted rooms. notify != highlight

Reply:
In reality if we wish to maintain the highlight behaviour this panel should probably be renamed to Mentions